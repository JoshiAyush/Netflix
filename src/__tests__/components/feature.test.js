import React from "react";
import { render } from "@testing-library/react";

import { Feature } from "../../components/index.js";

describe("<Feature />", () => {
    it("renders the <Feature /> with populated data", () => {
        const { container, getByText } = render(
            <Feature />
        );

        expect(getByText("Unlimited films, TV programmes and more.")).toBeTruthy();
        expect(getByText("Watch anywhere. Cancel at any time.")).toBeTruthy();

        expect(container.firstChild).toMatchSnapshot();
    });
});