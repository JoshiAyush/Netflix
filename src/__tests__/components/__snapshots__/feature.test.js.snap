// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Feature /> renders the <Feature /> with populated data 1`] = `
<div
  class="style__Container-jsps71-0 fwuSFk"
>
  <h1
    class="style__Title-jsps71-1 keyHoD"
  >
    Unlimited films, TV programmes and more.
  </h1>
  <h2
    class="style__SubTitle-jsps71-2 gkfDvA"
  >
    Watch anywhere. Cancel at any time.
  </h2>
  <div
    class="style__Container-aw8769-0 emHGMr"
  >
    <input
      class="style__Input-aw8769-1 jcPedt"
      placeholder="Email address"
    />
    <button
      class="style__Button-aw8769-2 jJrQMn"
      width="32.8%"
    >
      Try it now
       
      <img
        alt="Try Now"
        src="/images/icons/chevron-right.png"
      />
    </button>
    <p
      class="style__Text-aw8769-3 dicfXI"
    >
      Ready to watch? Enter your email to create or restart your membership
    </p>
  </div>
</div>
`;
